### DISCUSSION TITLE
Don't show "Refresh", but instead just reload the page

### DISCUSSION THREAD

**Discussion by tomayac** [community/community#4434] [Other Feature Feedback, Questions, & Ideas]
I dislike when GitHub _knows_ that its information is stale after a commit and it just displays the **Refresh** button. It could instead simply refresh itself, I don't mind the reload, but I do mind my time having to comment again because my proposals were made for outdated code.

![E3gsO7LXMAMwuRq](https://user-images.githubusercontent.com/145676/124080483-b7bd3e80-da4a-11eb-81b0-008b3c0b83d0.jpeg)

**Comment by gnprice** [comment]
This is indeed an annoying problem, but I think it's trickier than that to solve, and the proposed solution would not be an improvement.

The trouble is that if I'm in the middle of writing a comment, it can be pretty disruptive to have the page reload out from under me. What happens to the comment I was writing?

It seems like the core problem here is:
> having to comment again because my proposals were made for outdated code.

that the "Refresh" button can be pretty subtle when you're focused on writing a comment, and so you keep writing it without getting effectively notified that the code you're looking at is outdated.

Given that, perhaps a useful change would be to make that information prominent right there in the writing-a-comment box -- like right under the text input, next to the submit button.
